HW_5/Task1.py

```diff
@@ -0,0 +1,15 @@
+# Задача 26:  Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+
+# *Пример:*
+
+# A = 3; B = 5 -> 243 (3⁵)
+#     A = 2; B = 3 -> 8 
+
+# a = int(input('Введите первое число : '))         <-- Можно вводить и через запрос.
+# b = int(input('Введите второе число : '))
+
+def stepen (a, b):
+    if b == 1:
+        return (a)
+    return(a * stepen(a, b-1))
```

Comment:
Скобки мне нужны, `return` ведь это не функция/
```suggestion
        return a
    return a * stepen(a, b - 1)
```